apps/generator/lib/generator.js

```
@@ -864,37 +888,77 @@
+    let status;
+    if (matchedConditionPath) {
+      const { subject, validation } = this.templateConfig.conditionalGeneration[matchedConditionPath];
+      const parameterValue = await this.getParameterValue(asyncapiDocument, subject);
+
+      if (!parameterValue) return log.debug(logMessage.relativeSourceFileNotGenerated(relativeSourceFile, this.templateConfig.conditionalFiles[relativeSourceFile].parameter));
+    
+      const ajv = new Ajv();
+      const validate = ajv.compile(validation);
+      let isValid = validate(parameterValue);
+
+      if (validation.hasOwnProperty('not')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)